internal/client/integrations/integrations.go

```diff
@@ -1455,9 +1455,9 @@ func getIntegrationConnection(connectionName eventparameter,
 	ic.Version = strings.Split(*connectionVersion.Value.StringValue, "/")[9]
 	connectionType := strings.Split(*connectionVersion.Value.StringValue, "/")[5]
 	if connectionType == "gcp" {
-		ic.CustomConnection = false
-	} else {
 		ic.CustomConnection = true
```

Comment:
I don't think this is right. If the connection type is gcp, then it is NOT a custom connector. Please take a look at this PR (#221 ). I made some changes to this.

Reply:
Oh okay, I was getting an issue with the CLI for a connector. I did not realize that the connectors was a custom connector. I think the PR is not needed. I will try to debug what is issue and raise a new PR Thanks.